ports/atmel-samd/mpconfigport.h

```diff
@@ -54,6 +54,7 @@
 #define MICROPY_PY_DESCRIPTORS      (1)
 #define MICROPY_PY_MATH             (0)
 #define MICROPY_PY_CMATH            (0)
+#define MICROPY_PY_MICROPYTHON_MEM_INFO (0)
```

Comment:
Do you want to disable across the port, or only for M0? Granted, I imagine few people utilize the full stack info...

Reply:
I did the port to minimize inconsistencies. Hopefully the livestream helped show the alternative ways to debug.

Reply:
Sounds good to me!